### DISCUSSION TITLE
Fzf sorting after reloading by `--bind reload:change`

### DISCUSSION THREAD

**Discussion by Sangdol** [junegunn/fzf#3045] [Q&A] [answered]
Hello, 

Thanks for the great tool.

Is there any way to sort the result provided by a reload command?

The [coc-fzf](https://github.com/antoinemadec/coc-fzf) project reloads symbols through `--bind reload:change` ([code](https://github.com/antoinemadec/coc-fzf/blob/812b0a47d0c8b066684d75bc6f9ad84627f8579e/autoload/coc_fzf/symbols.vim#L71)) for each query since it's not possible to take all symbols from the languageserver and filter it using FZF.

The thing is that the result from the languageserver is not sorted, so the feature feels incomplete ([related ticket](https://github.com/antoinemadec/coc-fzf/issues/121)).

Ideally, we could reuse [the sophisticated fzf sorting](https://github.com/junegunn/fzf/blob/master/src/algo/algo.go) instead of implementing its own sorting algorithm.

I've checked out the documents, and there doesn't seem to be a way to achieve this, but I'm giving it a shot hoping to get at least some hints about this.

Thanks.

**Comment by junegunn** [comment, marked as answer]
Would it help to remove `--phony` option from the code?

https://github.com/antoinemadec/coc-fzf/blob/812b0a47d0c8b066684d75bc6f9ad84627f8579e/autoload/coc_fzf/symbols.vim#L72

**Reply by Sangdol** [reply]
It works perfectly without `--phony`.

It's embarrassing that I missed this simple option while saying I read documents.

That said, I'm glad that I asked it here before I go down the rabbit hole.

Thanks a lot for the quick and helpful answer.
